pegasus_wrapper/workflow.py

```diff
@@ -47,16 +52,15 @@ class WorkflowBuilder:
     created_by: str = attrib(validator=instance_of(str), kw_only=True)
     _workflow_directory: Path = attrib(validator=instance_of(Path), kw_only=True)
     _namespace: str = attrib(validator=instance_of(str), kw_only=True)
+    _default_site: str = attrib(validator=instance_of(str), kw_only=True)
```

Comment:
Why was the order changed?

Reply:
@LichMaster98

Reply:
I think mypy was complaining about the order because it doesn't know that non-default params that are KW only can go after ones with defaults. This can be returned to the previous order.